TradeItIosTicketSDK2/TradeItTradingTicketViewController.swift

```diff
@@ -25,6 +26,13 @@ class TradeItTradingTicketViewController: TradeItViewController, UITableViewData
     private var equityOrderCapabilities: TradeItInstrumentOrderCapabilities?
 
     private var selectedAccountChanged: Bool = true
+    
+    private lazy var updateQuotePrice: (TradeItQuote) -> Void = { quote in
+        self.quote = quote
+        self.order.quoteLastPrice = TradeItQuotePresenter.numberToDecimalNumber(quote.lastPrice)
+        self.reload(row: .marketPrice)
+        self.reload(row: .estimatedCost)
```

Comment:
Is there a way to update the price label without updating the entire row? The fading animation of the whole row is a little distracting.

Reply:
I added optional parameter `reload(row: TicketRow, animation: UITableViewRowAnimation = .automatic)`. So if you call it with `.none` like in this example `self.reload(row: .marketPrice, animation: .none)` you will not see the fading animation. It looked good when the price only changed will try to see how it will look if other part of the row changed.